app/models/measure_condition_augmenter.rb

```diff
@@ -0,0 +1,58 @@
+class MeasureConditionAugmenter
```

Comment:
This is too specific in naming, now. It has one of 3 modes (default, replace, augment). It's also not a model in the sense that it's a verb, not a noun.

Reply:
```suggestion
class MeasureConditionDialogOption
```